docs/content/latest/benchmark/tpcc-ysql.md

```diff
@@ -43,90 +46,79 @@ $ cd tpcc
 The binaries are compiled with JAVA 13 and it is recommended to run these binaries with that version.
 {{< /note >}}
 
-## Step 2. Start your database
+### Step 2. Start your database
 
-Start the database using the steps mentioned [here](https://docs.yugabyte.com/latest/quick-start/create-local-cluster/macos/).
+Start your YugabyteDB cluster by following the steps [here](../../deploy/manual-deployment/).
 
-## Step 3. Configure connection properties
+{{< tip title="Tip" >}}
+You will need the IP addresses of the nodes in the cluster for the next step.
+{{< /tip>}}
 
-Set the following connection configurations in the workload configuration file (`config/workload_all.xml`).
+### Step 3. Running the TPC-C benchmark
```

Comment:
Lets make this Step 2, and call it "TPC-C Load Phase"

Reply:
Done.